docs/design/autoware-interfaces/components/sensing.md

```diff
@@ -0,0 +1,46 @@
+# Sensing
+
+![Node diagram](images/Sensing-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+### The sensing module contains the driver for the sensor, so there is no input
+
+## Output
+
+### Camera image
+
+Image data from camera. Used by the Perception.
+
+- [sensor_msgs/Image](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/Image.html)
+
+### Lidar original pointcloud
+
+Pointcloud without preprocessing. Used by the Perception.
+
+- [sensor_msgs/PointCloud2](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/PointCloud2.html)
+
+### Lidar filtered pointcloud
+
+Pointcloud after preprocessing. Used by the Perception.
+
+- [sensor_msgs/PointCloud2](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/PointCloud2.html)
+
+### Radar pointcloud
```

Comment:
Thanks for your PR. Is this description based on any current implementation?

@scepter914 mentioned the radar interface in this discussion and proposed using RadarScan/RadarTracks.
https://github.com/orgs/autowarefoundation/discussions/2531

Reply:
At first, I wrote it based on the input and output of all nodes in the sensing module. RadarScan/RadarTracks should be sent by the driver,  which is converted to pointcloud by RadarScanToPointcloud2Node in the sensing module, so I didn't write this topic into the output of the sensing module. 
Actually, I'm not sure which one to use

Reply:
@beginningfan Sorry for my late reply.
As you said, there is also a Pointcloud2 interface, but in the current implementation, RadarTracks are input to the perception layer for fusing with LiDAR detected objects.
https://github.com/autowarefoundation/autoware.universe/blob/main/launch/tier4_perception_launch/launch/object_recognition/detection/lidar_radar_based_detection.launch.xml
https://github.com/autowarefoundation/autoware.universe/tree/main/perception/radar_tracks_msgs_converter

How about adding a RadarTracks interface in parallel to Pointcloud2 interface?

Reply:
Same as what @scepter914 said in discussion. :+1: 
https://github.com/orgs/autowarefoundation/discussions/2531#discussion-4173490

Reply:
I have added RadarTracks, do I need to replace pointcloud with RadarScan? 
[#discussion-4173490](https://github.com/orgs/autowarefoundation/discussions/2531#discussion-4173490), there are only RadarScan and RadarTracks in the summary, but there are often pointclouds in the following diagrams.

Reply:
@beginningfan Thanks!

> do I need to replace pointcloud with RadarScan?

Currently, the implementation status in the perception layer is as follows.
- RadarTracks: used by converting to TrackedObjects inside the perception layer
- RadarScan: currently not used
- Pointcloud2: no specific algorithm for RADAR Pointcloud, but can be commonly used as input to perception layer

For the above reasons, we think RadarTracks and pointcloud are good interfaces for sensing -> perception in the current status.